argoverse/evaluation/stereo/eval.py

```diff
@@ -0,0 +1,208 @@
+# <Copyright 2021, Argo AI, LLC. Released under the MIT license.>
+"""Stereo evaluation for the Argoverse stereo leaderboard.
+
+Evaluation:
+
+    We consider the disparity of a pixel to be correctly estimated if the absolute disparity error is less than a
+    threshold and its relative error is less than 10% of its true value, similar to the KITTI Stereo 2015 benchmark [1].
+    We define three disparity error thresholds: 3, 5, and 10 pixels.
+
+    Some stereo matching methods such as Semi-Global Matching (SGM) might provide sparse disparity maps, meaning that
+    some pixels will not have valid disparity values. In those cases, we interpolate the predicted disparity map using
+    a simple nearest neighbor interpolation as in the KITTI Stereo 2015 benchmark [1] to assure we can compare it to
+    our semi-dense ground-truth disparity map. Current deep stereo matching methods normally predict disparity maps
+    with 100% density. Thus, an interpolation step is not needed for the evaluation.
+
+    The disparity errors metrics are the following:
+
+    1. all: Percentage of stereo disparity errors averaged over all ground truth pixels in the reference frame
+            (left stereo image).
+    2. bg: Percentage of stereo disparity errors averaged only over background regions.
+    3. fg: Percentage of stereo disparity errors averaged only over foreground regions.
```

Comment:
maybe you could add one line explaining about how you computed the foreground region boundary?